### PR TITLE
Meta tags added

### PR DESCRIPTION
**Pull request by Avinash905**
## Related Issue

Closes: #187 

## Description of Changes

Added various meta tags like description, keywords, open graph, etc.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -2,11 +2,39 @@
...
 	<meta name="theme-color" content="#000000" />
-	<meta name="description" content="Web site created using create-react-app" />
+	<meta name="msapplication-TileColor" content="#da532c" />
+	<meta name="theme-color" content="#ffffff" />
+	<meta name="title" content="Grab Bits" />
+	<meta name="description" content="Grabbit is a community-driven platform that connects students with verified job, internship, and hackathon opportunities. It also offers motivational content from experienced professionals to inspire and guide students in their careers." />
+    <meta name="language" content="English">
+    <meta name="keywords" content="Grabbit, community-driven, platform, job opportunities, internship opportunities, hackathon, verified, students, skills, interests, experienced professionals, podcasts, motivational content, career development.">
+
+    <!-- Open Graph / Facebook -->
```
Comment (Grab-Bits) [inline client/public/index.html:18]: Hey @Avinash905 , we aren't on facebook for now can you please add our youtube, instagram , Linkedin and Whatsapp in its place !!

Comment (Avinash905) [reply client/public/index.html:18]: Open Graph meta tags are primarily used to control how shared links appear on social media platforms, including Facebook. However, they are not limited to Facebook alone. Other platforms like LinkedIn, Pinterest, and many others also support Open Graph meta tags.
Even if you don't have a Facebook account to share your content specifically on Facebook, including Open Graph meta tags in your website's HTML can still benefit you by ensuring that shared links on other social media platforms are displayed with accurate and appealing information.

**Comment by Bhushan8177** [comment]
Great job @Avinash905 , Merging this !!

**Comment by Bhushan8177** [comment]
Thank you for Contributing to GrabBit!!
Please ⭐ the repository and follow me on Github ...!

You can connect with me on Linkedin - 
https://www.linkedin.com/in/bhushan-kale/

**Pull request merged**